Cypress/cypress/integration/Parallel_Tests/Create_Workflow/preDefinedWorkflow.spec.js

```diff
@@ -69,11 +91,40 @@ describe("Testing the workflow creation wizard using PreDefined Experiments", ()
 					.eq(2)
 					.should("include.text", workflows.nonRecurringworkflowName); // Matching Workflow Name Regex
 				cy.wrap($div).find("td").eq(3).should("have.text", "Self-Agent"); // Matching Target Agent
+				// Workflow Statistics (Graph View)
+				cy.wrap($div).find("td").eq(2)
+					.invoke('attr', 'style', 'position: absolute')
+					.should('have.attr', 'style', 'position: absolute');
+				cy.wrap($div).find("td").eq(2).click();
+			});
+		// Wait for complete nodes to load
+		cy.waitUntil(() =>
```

Comment:
Can we have a more better check here? May be visiblity checks or no. of nodes check?